.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: java
 jdk:
   - oraclejdk8
-script: mvn clean verify
+script: "mvn cobertura:cobertura"
```

Comment:
@ZhijunZhao Before this command, Travis will run `mvn install` automatically. It will cover the `verify` phase.